### DISCUSSION TITLE
Track map: show flags on the track or corners

### DISCUSSION THREAD

**Discussion by BeyondVertical** [f1multiviewer/issue-tracker#198] [Feature requests] [answered]
It would be very cool to have the yellow sectors marked yellow on the track itself. I think they do it this way in the broadcast too.
For SC or VSC the whole track would be yellow.

**Comment by f1multiviewer** [comment, marked as answer]
This will be available in the next version! https://twitter.com/f1multiviewer/status/1576326597321592832
